WORKSPACE

```diff
@@ -1,5 +1,36 @@
 workspace(name = "org_iota_entangled")
 
+SNAPSHOT_REPOSITORY = "https://raw.githubusercontent.com/thibault-martinez/snapshots/master"
```

Comment:
Could you move these into `tools/snapshot.bzl` and just load/execute from WORKSPACE so we don't clutter this file too much?